### ISSUE TITLE
brew tap mitchellh/gon has stopped working

### ISSUE THREAD

**Issue by ndw** [mitchellh/gon#56]
I think this worked a couple of weeks ago, but I had a hardware problem and had to reinstall after repair. Now I get:

```
$ brew tap mitchellh/gon
==> Tapping mitchellh/gon
Cloning into '/usr/local/Homebrew/Library/Taps/mitchellh/homebrew-gon'...
remote: Enumerating objects: 12, done.
remote: Counting objects: 100% (12/12), done.
remote: Compressing objects: 100% (9/9), done.
remote: Total 12 (delta 2), reused 10 (delta 1), pack-reused 0
Receiving objects: 100% (12/12), done.
Resolving deltas: 100% (2/2), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/mitchellh/homebrew-gon/gon.rb
gon: Calling bottle :unneeded is disabled! There is no replacement.
Please report this issue to the mitchellh/gon tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/mitchellh/homebrew-gon/gon.rb:6

Error: Cannot tap mitchellh/gon: invalid syntax in tap!
```

**Comment by santiagocanti** [comment]
Same issue here, thanks for reporting

**Comment by mountaindude** [comment]
Looks like other brew packages have troubles too.
Found [this issue](https://github.com/openresty/openresty/issues/814) in another repo.

Downgrading brew worked for me:

```bash
pushd .
cd /usr/local/Homebrew
git fetch --tags
git checkout 3.3.9
popd
brew tap mitchellh/gon
brew install mitchellh/gon/gon
```

Not ideal but maybe an acceptable workaround for now.

**Comment by louistb** [comment]
+1 same here, sending out good energy and hoping for a fix

**Comment by louistb** [comment]
Update : i download the binary from the release section and it works just fine 👍 its not added to your bash profile, you need to call the binary directly but its good work around for now

**Comment by marcus-crane** [comment]
@mitchellh

**Comment by gandarez** [comment]
+1

**Comment by gandarez** [comment]
Actually the repository for homebrew to work is [this one](https://github.com/mitchellh/homebrew-gon). Looking at the commit history it hasn't changed since Mar 6, 2020. Possibily a change in Homebrew core might affected this repo?

**Comment by gandarez** [comment]
The tap is generated and uploaded by `goreleaser` [here](https://github.com/mitchellh/gon/blob/master/.goreleaser.yml#L51).

**Comment by gardc** [comment]
Welp. A workaround for CI:
```
brew install wget
wget https://github.com/mitchellh/gon/releases/download/v0.2.3/gon_macos.zip
unzip ./gon_macos.zip
rm -rf ./gon_macos.zip
chmod +x ./gon
```
Remember to call gon by its path e.g. `./gon`.

**Comment by mitchellh** [comment]
Hello! I'm sorry about this. I'll update Goreleaser and make another release. An issue is that I don't have any of my secrets setup on my Mac to be able to sign this release (the irony) since I switched to Linux. So I'm going to have to set that all up again 😄

**Comment by marcus-crane** [comment]
Gon isn't signed with Gon? 😏

**Comment by mitchellh** [comment]
It is but I don't have any of the certs setup, xcode installed, etc. Haha. I'm working on it now.

**Comment by mitchellh** [comment]
Fixed in 0.2.5
